dev/core/src/com/google/gwt/dev/jjs/impl/DeadCodeElimination.java

```diff
@@ -1996,6 +2021,7 @@ private Object tryTranslateLiteral(JExpression maybeLiteral, Class<?> type) {
    * labeled break break to another context.
    */
   public static class FindBreakContinueStatementsVisitor extends JVisitor {
+    // TODO add yield support?
```

Comment:
I don't think so. Looks like it is used with `do...while` and switch statements. continue/break can only occur in a switch expression case body within loops or switch statements. In both cases the switch expression case is not terminated. So I think if we ever need to search for `yield` we probably create a new visitor.

Reply:
I think you're right, but hadn't made the argument quite as convincingly to myself as you did.